students/BRACH_Jakub/lesson_01_basics/L01P01_Three_Numbers.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+number_of_factors = 3
+summ = 0
+
+for x in range(0, number_of_factors):
+   import math
+   print('Enter the number {0:d} of {1:d}:'.format(x, number_of_factors))
+   summ = summ + input()
```

Comment:
This line throws an error : `unsupported operand type(s) for +: 'int' and 'str' `
Need to cast str to int: `int(input())`

Reply:
Done